packages/next/test/fixtures/00-i18n-support/index.test.js

```diff
@@ -29,9 +31,16 @@ async function checkForChange(url, initialValue, hardError) {
 const ctx = {};
 
 describe(`${__dirname.split(path.sep).pop()}`, () => {
-  it('should deploy and pass probe checks', async () => {
+  beforeAll(async () => {
```

Comment:
It's not a bug, it's just that the first test creates the deployment and the other tests perform some assertions on the behavior of the deployment.

This `index.test.js` file is afaik copy/pasted into all the other `next/test/fixtures` directories so if we're making a change here then it should probably apply to all of them.

Reply:
I would consider this a bug because it relies on test order to execute properly.

I reverted the change here and will make a change in all of these files in a separate PR.